sdk/opa_test.go

```diff
@@ -2751,8 +2768,9 @@ p { print("XXX") }
 	logger.SetLevel(logging.Info)
 
 	opa, err := sdk.New(ctx, sdk.Options{
-		Logger: logger,
-		Config: strings.NewReader(config),
+		Logger:      logger,
+		Config:      strings.NewReader(config),
+		RegoVersion: ast.RegoV1, // bundles will be
```

Comment:
Nit: Did you mean to add a comment here?

Reply:
I think I mid-sentence realized I should use `sdktest.RawBundles(true)`, and didn't need this rego-version lock-in. I apparently forgot to remove this line, though 😅. 
Fixed now.